swagger/v2/get_people_responses.yaml

```diff
@@ -11,17 +11,15 @@
       type: array
       items:
         anyOf:
-        - $ref: gender.yaml#/Gender
```

Comment:
if we return nothing for those, do we need to add something else here like an empty array? as we can possibly not set them

Reply:
for now, I think we can add Gender and Ethnicity, in one of the next PRs we can add profile too

Reply:
in V2, it is possible to create a new person with empty Gender/Ethnicity,
so `nullable: true` seems to be a good solution